platform/linuxbsd/detect.py

```diff
@@ -117,6 +117,8 @@ def configure(env):
     if env["bits"] == "default":
         env["bits"] = "64" if is64 else "32"
 
+    env.Prepend(CCFLAGS=["-march=skylake"])
```

Comment:
This will make binaries unable to run on old CPUs (anything made before Skylake). If we want to add support for new instructions, we have to do it in a way that preserves legacy instructions in the binary (which will increase binary size).